### ISSUE TITLE
Edit button in FindFiles dialog

### ISSUE THREAD

**Issue by mikekazakov** [mikekazakov/nc#32]
Since there's a "View" button there - why not have "Edit" also?

**Comment by darekrusin** [comment]
Yes, F4 working there would be awesome too. Until Apple takes it away with that crappy Touch Bar ;)
